### PR TITLE
Add function "getFeatureType()"

### LINKED ISSUE #24: Add function to query FeatureType
A function `getFeatureType` should be added to the `LayerClient` in order to get detailed information about a FeatureType.

Corresponding URL:
`http://localhost:8080/geoserver/rest/workspaces/<my-workspace>/datastores/<my-datastore>/featuretypes/<my-featuretype>.json`

### PR DESCRIPTION
**Pull request by JakobMiksch**
- fixes #24
- add function "getFeatureType()"

### CODE REVIEW

File: src/layer.js [hunk 1]
```diff
@@ -220,6 +220,41 @@ export default class LayerClient {
...
+      method: 'GET',
+      headers: {
+        Authorization: this.auth
+      }
+    });
+
+    if (!response.ok) {
+      const grc = new AboutClient(this.url, this.auth);
+      if (await grc.exists()) {
+        // GeoServer exists, but requested item does not exist,  we return empty
```
Comment (chrismayer) [inline src/layer.js:247]:
```suggestion
        // GeoServer exists, but requested item does not exist, we return empty
```

File: src/layer.js [hunk 2]
```diff
@@ -220,6 +220,41 @@ export default class LayerClient {
     }
   }
 
+  /**
+   * Get detailed information about a featureType.
```
Comment (chrismayer) [inline src/layer.js:224]: Pleas use `FeatureType` notation (with upper case). Also for the API doc notations below.

**Review by chrismayer — COMMENTED** [review]
2 minor things which should be adapted before merging.

File: src/layer.js (hunk 2 above)
Comment (JakobMiksch) [reply src/layer.js]: done with [8a70879](https://github.com/meggsimum/geoserver-node-client/pull/71/commits/8a7087989a8d9f88f8017a6554f9730a0a4faef1)

**Pull request merged**